src/main/webapp/calendar-script.js

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
-document.addEventListener('DOMContentLoaded', function() {
+// Triggered upon DOM load.
+$(document).ready(() => {
```

Comment:
This is only personal preference, as I used jQuery elsewhere for this use. I'm happy to change this back if wanted.

Reply:
as long as it doesn't change functionality (it shouldn't) then I'm good with this. I dont have a preference